pages/index.tsx

```diff
@@ -17,3 +23,27 @@ export default function Component() {
         </>
     );
 }
+
+/**
+ * getServerSideProps runs before each page is rendered to check to see if a
+ * user has already been authenticated
+ */
+export async function getServerSideProps(context: GetSessionOptions) {
+    // obtain the next auth session
+    const session = await getSession(context);
+
+    // if the user is not authenticated redirect them to the login page
```

Comment:
Should we be redirecting them to the login page or the home page?

Reply:
That's what I was wondering at first as well, but if our platform is specifically for the class management stuff then I would assume you need to log in to be able to access anything? Or are you allowed to browse programs even if you're not registered?

Reply:
You're allowed to browse programs even if you're not logged in.

Reply:
It's something we discussed in May/June during our team meetings